source/configuration_manual/notify_status_plugin.rst

```diff
@@ -35,14 +35,24 @@ Example
 
   mail_plugins = $mail_plugins notify notify_status
 
-  plugin {
-    notify_status_dict = proxy:dict-async:notify_status
+  notify_status {
+    dict_driver = proxy
+    dict_proxy_name = notify
```

Comment:
* Originally it was dict_proxy_name = notify_status
 * Add dict_proxy_socket_path = dict-async also

Reply:
Fixed it